flake.nix

```diff
@@ -319,13 +319,21 @@
               gzip bzip2 lzma gnutar unzip git gitAndTools.topGit mercurial darcs gnused breezy
             ] ++ lib.optionals stdenv.isLinux [ rpm dpkg cdrkit ] );
 
+          # Can be uncommented when https://github.com/TJC/Test-postgresql/issues/40 gets fixed.
+          # YATH_JOB_COUNT = "\${NIX_BUILD_CORES-$(nproc)}";
+          YATH_JOB_COUNT = "4";
```

Comment:
Hmm... We probably don't want to force parallel job running for everybody: some users will definitely want to run not in parallel. Maybe we should have it be not-parallel by default, and document -j? Not sure exactly. Maybe more specifically we'd want it to be `YATH_JOB_COUNT=max(4, NIX_BUILD_CORES, nproc)`

Reply:
I think I'll drop this altogether, in favor of updating the hacking docs to note the ability to run tests in parallel (with a sidenote that they may fail spuriously, with a link to that issue).